File: src/main/java/infinitespire/relics/BlanksBlanky.java
package infinitespire.relics;

import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
import com.megacrit.cardcrawl.relics.AbstractRelic;

import infinitespire.InfiniteSpire;
import infinitespire.abstracts.Relic;

public class BlanksBlanky extends Relic {

	public static final String ID = InfiniteSpire.createID("Blanks Blanky");
	
	public BlanksBlanky() {
		super(ID, "blanksblanky", RelicTier.SPECIAL, LandingSound.MAGICAL);
	}

	@Override
	public AbstractRelic makeCopy() {
		return new BlanksBlanky();
	}

	@Override
	public void onRest() {
		InfiniteSpire.logger.info("InfiniteSpire | Healing player with Blank's Blanky.");
		AbstractDungeon.player.currentHealth = AbstractDungeon.player.maxHealth;
	}
}

File: src/main/kotlin/com/blanktheevil/infinitespire/relics/BlanksBlanky.kt
package com.blanktheevil.infinitespire.relics

import com.blanktheevil.infinitespire.extensions.makeID
import com.blanktheevil.infinitespire.extensions.player

class BlanksBlanky : Relic(ID, IMG, TIER, SOUND) {
  companion object {
    val ID = "Blanks Blanky".makeID()
    private val IMG = "blanksblanky"
    private val TIER = RelicTier.SPECIAL
    private val SOUND = LandingSound.MAGICAL
  }

  override fun onRest() {
    player.currentHealth = player.maxHealth
  }
}